eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -6,10 +6,12 @@ parameters:
   EnvVars: {}
   MaxParallel: 0
   TimeoutInMinutes: 60
+  PublishCodeCoverage: false
   Matrix:
     Linux_Node10:
       OSVmImage: "ubuntu-16.04"
       TestType: "node"
+      PublishCodeCoverage: true
```

Comment:
How did you choose which set of test legs to publish tests for?

Reply:
so I chose one for node and one for browser, and preference given to node 10 (amongst 8,10 and 12 if present) and Linux (among mac, windows and linux)